hypothesis-python/docs/community.rst

```diff
@@ -45,3 +45,6 @@ As well as being beautiful, dragonflies actively hunt down bugs for a living!
 You can find the images and a usage guide in the :gh-file:`brand` directory on
 GitHub, or find us at conferences where we often have stickers and sometimes
 other swag.
+
+Stickers and :gh-file:`hard enamel contributor pins <brand/pins.jpg>` are also
+available by mail for `github sponsors <https://github.com/sponsors/Zac-HD/>`__.
```

Comment:
It is _really_ not clear from this phrasing that you mean sponsoring you instead of sponsoring Hypothesis. I'd rather you removed this line entirely from here.

Reply:
Rereading that, yeah - definitely not my intention.  Removed.